src/service/repo/anonymousMessageRepo.ts

```diff
@@ -31,6 +34,7 @@ interface AnonymousMessageRepoTypes {
   getAllAnonymousPostNotifications: () => Promise<AnonymousPostNotification[]>;
   getSingleAnonymousPostNotifications: (activityId: string) => Promise<AnonymousPostNotification>;
   setChannelAsRead: (channelId: string) => Promise<boolean>;
+  getAnonymousChannelDetail: (channelType: GetstreamChannelType, channelId: string) => Promise<any>;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The return type of `getAnonymousChannelDetail` is `Promise<any>`. It's generally a good practice to avoid using `any` in TypeScript because it defeats the purpose of static typing. If possible, replace `any` with a more specific type.
```diff
- getAnonymousChannelDetail: (channelType: GetstreamChannelType, channelId: string) => Promise<any>;
+ getAnonymousChannelDetail: (channelType: GetstreamChannelType, channelId: string) => Promise<SpecificType>;
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->